frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/RebaseOnParentBackgroundable.java

```diff
@@ -0,0 +1,185 @@
+package com.virtuslab.gitmachete.frontend.actions.base;
```

Comment:
We are getting a Lombok `HandleExtensionMethod` error for this file. The error seems to point to some place where it is asserted that the type annotation should not be null. Any ideas?

Reply:
sorry for the late reply. I ve already looked into it but I did not find anything yet

Reply:
Thanks for looking into this. The error message is not very descriptive, either.

Reply:
ok, I have a solution but I don't see the problem XD 

conditions: gradle task compileKotlin
problem: ?
solution: drop shouldOverrideForkPoint just like I proposed below

Reply:
pls confirm it works for you

Reply:
> class com.sun.tools.javac.code.Symbol$ClassSymbol cannot be cast to class javax.lang.model.element.ExecutableElement

Reply:
the error was not in the RebaseOnParentBackgroundable

Reply:
though we have a `rebaseable`  boolean there. I can try removing it in a similar way. This seems to be a Lombok bug,

Reply:
actually, I have encountered checker framework excpetion about nullable selectedCommit (not annotated). seem to be a lombok issue to me too. anyway let's stick to the working solution

Reply:
I annotated the `selectedCommit`. Still, the source of class cast exception remains unclear 😕

Reply:
I removed the `Array` and `GitVfsUtils` Extensions and it finally went away.